background/index.js

```diff
@@ -75,27 +75,38 @@ chrome.runtime.onMessage.addListener((request, sender, callback) => {
 
 let codeCache = null;
 function loadScripts (port) {
+  function checkResponseOk (response) {
+    if (response.ok) {
+      return response;
+    }
+    const error = new Error('Expected successful response. Likely a version mismatch.');
+    error.isVersionMismatch = true;
+    throw error;
```

Comment:
Might want to have a custom error class whether than a property added to the standard Error one.. ?